src/db/sysdb_search.c

```diff
@@ -374,7 +374,9 @@ errno_t sysdb_getpwuid_with_views(TALLOC_CTX *mem_ctx,
         DEBUG(SSSDBG_OP_FAILURE, "talloc_new failed.\n");
         return ENOMEM;
     }
-
+    if (uid > 20000 || uid < 0) {
```

Comment:
Why 20000 was chosen? Please, add a new #define with a meaningful name (or check if we already have one and use it) in order to have things more clear.